### PR TITLE
7.50.2 changelogs

### PR DESCRIPTION
**Pull request by kacper-murzyn**
Agent 6/7.50.2 bugfix release changelog updates.

### CODE REVIEW

File: CHANGELOG-DCA.rst [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
+
+7.50.2 / 6.50.2
+======
+
+.. _Release Notes_7.50.2_Bug Fixes:
+
+Bug Fixes
+---------
+
+- Fixes a bug introduced in `7.50.0` preventing `DD_TAGS` to be added to `kubernetes_state.*` metrics.
```
Comment (AliDatadog) [inline CHANGELOG-DCA.rst:15]: wasn't this in 7.50.1 ?

**Review by AliDatadog — APPROVED** [review]

**Review by Pythyu — APPROVED** [review]

File: CHANGELOG-DCA.rst (hunk 1 above)
Comment (clamoriniere) [reply CHANGELOG-DCA.rst:15]: yes

Comment (kacper-murzyn) [reply CHANGELOG-DCA.rst:15]: I see that his change is in 7.50.1 Agent changelog but no in DCA one. What should be the correct state?

Comment (kacper-murzyn) [reply CHANGELOG-DCA.rst:15]: I see now, that changelog entry was added later and probably it was incorrectly caught by the tooling. I can change the DCA changelog entry to 7.50.1 to make it consistent. Would that be ok?

**Pull request merged**